.env.example

```
@@ -1,3 +1,5 @@
+COMPOSE_FILE=docker-compose.yml:docker-compose.override.local.yml:docker-compose.override.dev.yml
```

```suggestion
COMPOSE_FILE=docker-compose.yml:docker-compose.override.local.yml:docker-compose.override.locall.yml
```

Note the recommended for local development is `local`, `dev` is for dev.qfield.cloud